src/neuroconv/tools/nwb_helpers/_configure_backend.py

```diff
@@ -21,4 +22,14 @@ def configure_backend(
 
         # TODO: update buffer shape in iterator, if present
 
-        nwbfile_objects[object_id].set_data_io(dataset_name=dataset_name, data_io_class=data_io_class, **data_io_kwargs)
+        if isinstance(nwbfile_objects[object_id], Data):
+            nwbfile_objects[object_id].set_data_io(data_io_class=data_io_class, data_io_kwargs=data_io_kwargs)
+        elif isinstance(nwbfile_objects[object_id], TimeSeries):
```

Comment:
What happens with the timestamps if they are present?  Are they chunked compressed?

Reply:
Ought to be

Reply:
Are they `Data` or a `TimeSeries`? I guess that was my question.

Reply:
They are a dataset of the TimeSeries

Reply:
(it would enter line 29 here with `dataset_name="timestamps"`)